contracts/VotingRewards.sol

```diff
@@ -460,15 +459,20 @@ contract VotingRewards is AragonApp {
         address _beneficiary,
         uint64 _fromBlock,
         uint64 _toBlock
-    ) internal view returns (uint256) {
+    ) internal view returns (uint256, bool) {
         uint256 missingVotes = 0;
-        uint256 minimumBalance = 0;
-        uint256 votesLength = dandelionVoting.votesLength();
         uint64 voteDurationBlocks = dandelionVoting.durationBlocks();
-        bool isFirstUserVoteInEpoch = true;
 
-        // voteId starts from 1 in DandelionVoting
-        for (uint256 voteId = votesLength; voteId >= 1; voteId--) {
+        // NOTE: balance at the end of an epoch
+        uint256 minimumBalance = MiniMeToken(dandelionVoting.token()).balanceOf(
+            _beneficiary
+        );
+
+        for (
+            uint256 voteId = dandelionVoting.votesLength();
```

Comment:
`voteId` should become `voteProposalId` since it's not the ID of a vote (that a user has cast) but the ID of an entire voting proposal, right?